src/services/filter/filters/evm/filter.rs

```diff
@@ -297,7 +320,11 @@ impl<T> EVMBlockFilter<T> {
 														args: Some(params.clone()),
 														hex_signature: Some(format!(
 															"0x{}",
-															hex::encode(function.short_signature())
+															hex::encode(
+																&keccak256(
+																	function.signature().as_bytes()
+																)[..4]
+															)
 														)),
```

Comment:
Above you're using `selector` and here you're constructing the hash from `signature`, can we not use `selector` here as well?